homework-classes/ContributorsView.js

```diff
@@ -19,8 +19,46 @@
      * @param {Object[]} contributors An array of contributor objects
      */
     render(contributors) {
-      // TODO: replace this comment and the console.log with your own code
       console.log('ContributorsView', contributors);
```

Comment:
Remember to take out debug-type console.log statements out of production code (which a PR always is).